spec/integration/specs/app_spec.ts

```diff
@@ -32,22 +48,58 @@ describe("app start", () => {
       return
     }
     await clickWhatsNew(app)
-    await app.client.click("a#add-cluster")
-    await app.client.waitUntilTextExists("legend", "Choose config:")
-    await app.client.selectByVisibleText("select#kubecontext-select", "minikube (new)")
-    await app.client.click("button.btn-primary")
+    await addMinikubeCluster(app)
     await app.client.waitUntilTextExists("pre.auth-output", "Authentication proxy started")
-    let windowCount = await app.client.getWindowCount()
-    // wait for webview to appear on window count
-    while (windowCount == 1) {
-      windowCount = await app.client.getWindowCount()
-    }
-    await app.client.windowByIndex(windowCount - 1)
+    await waitForWebview(app)
     await app.client.waitUntilTextExists("span.link-text", "Cluster")
     await app.client.click('a[href="/nodes"]')
     await app.client.waitUntilTextExists("div.TableCell", "minikube")
   })
 
+  it('allows to create a pod', async () => {
+    const status = spawnSync("minikube status", {shell: true})
+    if (status.status !== 0) {
+      console.warn("minikube not running, skipping test")
+      return
+    }
+    await clickWhatsNew(app)
+    await addMinikubeCluster(app)
+    await app.client.waitUntilTextExists("pre.auth-output", "Authentication proxy started")
+    await waitForWebview(app)
+
+    await app.client.waitUntilTextExists("span.link-text", "Cluster")
+    await app.client.click(".sidebar-nav #workloads span.link-text")
+    await app.client.waitUntilTextExists('a[href="/pods"]', "Pods")
+    await app.client.click('a[href="/pods"]')
+    await app.client.waitUntilTextExists("div.TableCell", "kube-apiserver-minikube")
+    await app.client.click('.Icon.new-dock-tab')
+    await app.client.waitUntilTextExists("li.MenuItem.create-resource-tab", "Create resource")
+    await app.client.click("li.MenuItem.create-resource-tab")
+    await app.client.waitForVisible(".CreateResource div.ace_content")
+    // Write deployment manifest to editor
+    await app.client.keys("apiVersion: apps/v1\n")
+    await app.client.keys("kind: Deployment\n")
+    await app.client.keys("metadata:\n")
+    await app.client.keys("  name: nginx-deployment\n")
+    await app.client.keys("\uE003spec:\n")
```

Comment:
need to press backspace to get yaml formatted properly on ace editor

Reply:
Should we put backspace to a constant?